packages/admin/admin/src/index.ts

```diff
@@ -50,6 +50,7 @@ export { CrudContextMenu } from "./dataGrid/CrudContextMenu";
 export { CrudVisibility, CrudVisibilityProps } from "./dataGrid/CrudVisibility";
 export { ExportApi, useDataGridExcelExport } from "./dataGrid/excelExport/useDataGridExcelExport";
 export { GridFilterButton } from "./dataGrid/GridFilterButton";
+export { MoreActionsDivider, MoreActionsGroup, MoreActionsMenu, SelectedItemsChip } from "./dataGrid/MoreActionsMenu";
```

Comment:
I think `SelectedItemsChip` shouldn't be exported but stay a private component of MoreActionsMenu

```suggestion
export { MoreActionsDivider, MoreActionsGroup, MoreActionsMenu } from "./dataGrid/MoreActionsMenu";
```

Reply:
https://github.com/vivid-planet/comet/pull/2115/commits/2fe7f376bd47dc9d34659fcfcfc0831c62295c08